src/AbstractCsv.php

```diff
@@ -190,7 +190,7 @@ protected function newInstance($class, $open_mode)
         $csv->delimiter = $this->delimiter;
         $csv->enclosure = $this->enclosure;
         $csv->escape = $this->escape;
-        $csv->encodingFrom = $this->encodingFrom;
+        $csv->input_encoding = $this->input_encoding;
```

Comment:
I was in the mood for snake case :) . Seriously I did this to match $input_bom and $output_bom which are already in snake case. I should probably review all the codebase and make a clear decision between both coding style. Of note none of them are in contradiction to PSR2 if I recall but yes the code needs consistency.